pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -58,6 +74,29 @@ pub mod pallet {
 	#[pallet::getter(fn organizer)]
 	pub type Organizer<T: Config> = StorageValue<_, T::AccountId, OptionQuery>;
 
+	#[pallet::storage]
+	pub type MintAvailable<T: Config> = StorageValue<_, bool, ValueQuery>;
+
+	#[pallet::type_value]
+	fn DefaultMintFee<T: Config>() -> BalanceOf<T> {
+		(1_000_000_000_000_u64 * 55 / 100).into()
+	}
+
+	#[pallet::storage]
+	pub type MintFee<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery, DefaultMintFee<T>>;
+
+	#[pallet::type_value]
+	fn DefaultMintCooldown<T: Config>() -> BlockNumberOf<T>
+	where
+		<T as frame_system::Config>::BlockNumber: From<i32>,
+	{
+		5.into()
+	}
```

Comment:
This should work:
```suggestion
	pub fn DefaultMintCooldown<T: Config>() -> T::BlockNumber {
		5_u8.into()
	}
```

Reply:
The reason why `u8` works is because `frame_system::Config::BlockNumber` is bounded with [AtLeast32BitUnsigned trait](https://github.com/paritytech/substrate/blob/563f4820d8f36d256ada7ea3fef46b2e94c4cd5a/frame/system/src/lib.rs#L241) which implements `From` / `Into` conversions